File: 파이썬/행렬의곱셈.py
def solution(arr1, arr2):
    answer = [[0] * len(arr2[0]) for _ in range(len(arr1))]

    for i, r in enumerate(arr1):
        for j, c in enumerate(zip(*arr2)):
            answer[i][j] = sum([x*y for x, y in zip(r, c)])
    return answer

File: 자바스크립트/행렬의곱셈.js
function solution(arr1, arr2) {
    return arr1.map((row) => // 첫번째 배열 행의 갯수
        arr2[0].map((_, j) => // 두번째 배열 열의 갯수
            row.reduce((acc, num, i) => acc + num * arr2[i][j], 0)))
}